prepare/metrics/llama_index_corectness.py

```diff
@@ -0,0 +1,57 @@
+from unitxt import add_to_catalog
+from unitxt.metrics import LlamaIndexCorrectnessMetric
+from unitxt.test_utils.metrics import test_metric
+
+# metric = MetricPipeline(
+#     main_score="LlamaIndexCorrectness",
+#     # preprocess_steps=[
+#     #     CopyFields(field_to_field=[("references/0", "references")], use_query=True),
+#     #     CastFields(
+#     #         fields={"prediction": "float", "references": "float"},
+#     #         failure_defaults={"prediction": None},
+#     #         use_nested_query=True,
+#     #     ),
+#     # ],
+#     metric=LlamaIndexCorrectnessMetric(),
+#     postpreprocess_steps=[
+#         # CopyFields(
+#         #     field_to_field=[
+#         #         ("score/instance/score", "score"),
+#         #     ],
+#         #     use_query=True,
+#         # )
+#     ],
+# )
```

Comment:
```suggestion
```
why this endded up not working? Just curious

Reply:
`ast.literal_eval(references)`   where `references` is a list of two strings. but `literal_eval` expects `node_or_string`